browser/net/brave_site_hacks_network_delegate_helper.cc

```diff
@@ -35,7 +35,10 @@ namespace {
 
 const std::string& GetQueryStringTrackers() {
   static const base::NoDestructor<std::string> trackers(base::JoinString(
-      std::vector<std::string>({"fbclid", "gclid", "msclkid", "mc_eid"}), "|"));
+      std::vector<std::string>({"fbclid", "gclid", "msclkid", "mc_eid",
+                                "_hsenc", "__hssc", "__hstc", "__hsfp",
```

Comment:
I would be happy to also have an issue link here

Reply:
@iefremov Do you mean as  comment at the end of the line?

Something like this?

    "_hsenc", "__hssc", "__hstc", "__hsfp",  // brave/brave-browser#9019

Reply:
What works best for you.
I prefer either full links or `issues/9019` for ease of copypasting :)